packages/forms/src/Components/Concerns/HasHeaderActions.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace Filament\Forms\Components\Concerns;
+
+use Closure;
+use Filament\Forms\Components\Actions\Action;
+use Filament\Support\Enums\ActionSize;
+use Illuminate\Support\Arr;
+
+trait HasHeaderActions
+{
+    /**
+     * @var array<Action> | null
```

Comment:
This array could also contain action groups, right?

Reply:
if not mistaken currently Form and Infolist does not support action groups
https://filamentphp.com/docs/3.x/forms/actions
https://filamentphp.com/docs/3.x/infolists/actions

only Table has it
https://filamentphp.com/docs/3.x/tables/actions#grouping-actions

Reply:
Good point. I do think we need them for section actions to be useful to be honest, @danharrin?

Reply:
I think it would be quite a bit of work, lets hold off for now